community/best_practices_managing_issues.md

```diff
@@ -0,0 +1,53 @@
+# Best practices for managing onnx issues
+These best practices apply to all onnx ecosystem repos under SIG governance.
+
+## Enforce templates
+Currently there are no templates for creating issues in onnx. This makes it very difficult for community members to triage issues. Going forward templates shall be added and maintained for ONNX issues.
+
+Following templates shall be added
+* Report a bug
+* Ask a question
+* Propose a new feature, new or updated operator, or other enhancement
+
+
+## Label issues
+
+* Going forward all issues must be labeled. 
+* Multiple labels should be used to properly capture the issue. For example when a user logs a shape inference bug for a particular operator.  [bug][shape-inference] labels should be added.
+* Labels should use a consistent naming scheme
+
+### Rules for label naming scheme
+
+* Only lower case alphabets must be used
+* Whenever there are multiple words they must be separated by a hyphen '-'.
+* Names must be descriptive
+
+### Label tags
+
+Following labels will be added. [Some of these labels already exists]
+* bug
+* question
+* enhancement
+* good-first-issue
+* operators
+* build
+* documentation
+* test
+* spec
+* converters
+* version_converter
+* training
+* shape-inference
+* onnx-checker
+
+
+## Close abandoned issues
+
+Onnx shall use Probot/stale to close any abandoned issues.
```

Comment:
Since this is a guidelines doc, you should lead with the policy. What tool is used is secondary. If you specify a tool, you may also need to provide information on how each repo owner should set it up